tests/test_image_to_text_example.py

```diff
@@ -57,6 +57,7 @@ def _test_image_to_text(
         f"--model_name_or_path {model_name}",
         f"--batch_size {batch_size}",
         "--max_new_tokens 20",
+        "--use_kv_cache",
```

Comment:
Previous CI did not enable this, do we see similar perf without this first? I think clip doesnt use kv cache so it likely won't help that case.
Alternatively we can add also flash_attention options and see most optimized perf (if it aligns with README)